docs/pages/docs/constants/rpcs.en-US.mdx

```diff
@@ -2,13 +2,15 @@
 
 ## alchemyRpcUrls
 
-RPC URLs supported by [Alchemy](https://www.alchemy.com).
+RPC URLs supported by [Alchemy](https://alchemy.com/?a=wagmi).
```

Comment:
hope this is ok -- useful for us to track how many people are visiting from wagmi docs :)